contracts/ZecaleDispatcher.sol

```diff
@@ -118,56 +114,53 @@ contract ZecaleDispatcher
         // Create an array to reuse to pass the nested inputs to the
         // application (note that the result is not passed, hence -1).
         uint256[] memory nested_proof_inputs =
-            new uint256[](_inputs_per_nested_tx - 1);
+            new uint256[](_inputs_per_nested_tx);
+
+        // Result bits
+        uint256 results = inputs[3];
```

Comment:
Can you update the comment above describing the structure of the inputs (i.e. `// Format of `inputs` matches the proof inputs exactly: [...]`)? Seems like the fact that the aggregate result comes right after the vk digest words is not documented there (in fact, as documented, `inputs[3]` should return a chunk of `nested_inputs_1`)

Reply:
Updated the comment.  thanks.